### PR TITLE
Avoid cycle

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jul 19 08:10:14 UTC 2019 - Josef Reidinger <jreidinger@suse.com>
+
+- avoid dependency on yast2-installation
```
Comment (teclator) [inline package/yast2-network.changes]: A bug number will probably be needed

Comment (mchf) [reply package/yast2-network.changes]: also it is not dependency on yast2-installation

Comment (jreidinger) [reply package/yast2-network.changes]: well, it is not accepted to target project, it should work without it

Comment (teclator) [reply package/yast2-network.changes]: OK, so then fine with me

Comment (teclator) [inline package/yast2-network.changes]:
```suggestion
- avoid dependency on autoyast2-installation
```

**Pull request merged**